### ISSUE TITLE
when creating a 'same' property, is it possible for metadata to appear in parents?

### ISSUE THREAD

**Issue by crazymedguy** [SkepticMystic/breadcrumbs#244]
So from what I understand, the 'same' property means having an adjacent sibling under the same parent, but not related to with 'next' or 'previous'. This would mean that it shouldn't be an implied relationship, but a real relationship - something like this:

- Parent
  - Page A that you are editing with the same:: metadata
  - Page B that you are linking to with the same:: metadata

IMO when linking to page B while in page A, the parent page should also equally reflect Page B as a child automatically as a real child. What do you think?

**Comment by SkepticMystic** [comment]
Just to be sure, are you saying the real relationships (in black) should imply the relationships in red?
![image](https://user-images.githubusercontent.com/70717676/148635523-fa0bb877-26e0-48db-8e4c-fbdd419e9bc4.png)

**Comment by crazymedguy** [comment]
Yup that's right! Unless I misunderstood the hierarchy? Thought that was the case. I think it's currently reflected in the sidebar and as implied but I thought it should also have the same parent. I'm not too sure about "half-siblings" and how they'd process logically though.

**Comment by scriptor-dga** [comment]
I'm also very interested in this. I don't understand why implied children or parents don't show up in any view.

**Comment by SkepticMystic** [comment]
I will try to make this optional, as I'm sure not everyone wants this behaviour

**Comment by SkepticMystic** [comment]
@scriptor-dga @crazymedguy a more general discussion can be found in #259, I'm gonna close this issue to have all discussion focused there :)
